build.sbt

```diff
@@ -29,18 +29,22 @@ inThisBuild(
     ),
     licenses := Seq("Apache-2.0" -> url(s"${scmInfo.value.map(_.browseUrl).get}/blob/v${version.value}/LICENSE")),
     pgpPublicRing := file("/tmp/public.asc"),
-    pgpSecretRing := file("/tmp/secret.asc"),
-    releaseEarlyWith := SonatypePublisher
+    pgpSecretRing := file("/tmp/secret.asc")
```

Comment:
You need to set `pgpPassphrase` as well. See [here](https://github.com/zio/interop-monix/blob/master/build.sbt#L18).